### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by guumaster** [MostlyAdequate/mostly-adequate-guide#150]
Reopened from #149. Any thoughts/ideas on how to coordinate it?

**Comment by KtorZ** [comment]
I still think that using a version number for each chapter is a good (and quite simple) idea. Or maybe a changelog that references the commit on which is based the current translation.

**Comment by DrBoolean** [comment]
The version idea is a great one, though I'm thinking it might be overkill.

My thoughts are that the bulk of the PRs, grammatical changes, probably won't apply. If there's a way to capture and propagate the logic/code/explanation updates, that'd be optimal. I'm happy to maintain a changelog for any major updates or open to other ideas.

/cc @maksimf

**Comment by KtorZ** [comment]
What about small git hooks to analyse changes once they are pushed on the repository in order to, in a second time, generate changelogs automatically ? One file that summarise all changes made in the english version and not yet impacted in a given translated version. 

Plus, if we assign a number to each change, we could use commit messages to update it again automatically when submitting a new translation.. something simple, following what GitHub is doing for PR like "Translate #14 to FR"

**Comment by maksimf** [comment]
Do we really need to track all the small changes? Maybe we can approach this as paper books. When new edition of the book is ready, we translate it.

**Comment by guumaster** [comment]
Just in case this one is lost from the previous thread: 

> Looking around I've found the boot [Pro Git 2](https://progit.org/) that is managed [here](https://github.com/progit) as an organization, with a repo per language (look for the progit2-* repos). And apparently is later built with some scripts. 
> 
> Taking that one as example, what I propose is the following: 
> 
> * Create an organization with the main repo (english version).
> * Create a team to maintain the org daily operations, etc.
> * Create a repo for each translation requested (as issues on the main repo), and a language specific team with permissions over that repo.
> * Rinse/repeat last step per each requested language.

Organizations comes free, a repo per language with a upstream origin to merge english changes may not be too complicated.

**Comment by KtorZ** [comment]
Yep. I suggest to go for a simple solution so that we can start. 

@DrBoolean May you create the organisation and invite interested people ? 
So far, @maksimf for the russian version, @guumaster for the spanish, and me for the french version. 

We won't be able to fork the existing repository in the same organisation. However, we can easily keep the same structure as the english reference repository and I suggest to have a changelog file on each translated repository that associates to each chapter a commit of the ref english repo;

**Comment by llh911001** [comment]
I am the [Simplified Chinese](https://github.com/llh911001/mostly-adequate-guide-chinese) version translator, my suggestion is that create your own repo, do the translation, then add your repo's link in the original repo's [README.md](https://github.com/DrBoolean/mostly-adequate-guide#other-languages) and make a PR. Just a suggestion though :)

**Comment by guumaster** [comment]
**TL;DR**: *Do not modify original files to translate them, create a new one with your language suffix. *

I think it would be easier to merge future changes if every language repo keep the original file "as is". For example: 

```bash
ch1.md
ch1-ru.md
ch2.md
ch2-ru.md
```

So when @DrBoolean edits or merge any PR to the base version, language maintaners can keep up with it just by reading a diff.

**Comment by maksimf** [comment]
@guumaster good idea, thank you
